xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -349,15 +351,40 @@ void CGUIFontTTF::DrawTextInternal(CGraphicContext& context,
   Begin();
   uint32_t rawAlignment = alignment;
   bool dirtyCache(false);
+
+#if defined(HAS_GL)
+  // round coordinates to the pixel grid. otherwise, we might sample at the wrong positions.
+  if (!scrolling)
+    x = std::round(x);
+  y = std::round(y);
```

Comment:
What if texts scroll in the y-direction? I think it looks a little better this way but I can be fooled.
```suggestion
  if (!scrolling)
  {
    x = std::round(x);
    y = std::round(y);
  }
```

Reply:
It can't. Scrolling in this context means "single line of text moving from right to left" (or vice versa). Any vertical scrolling is done by animating the whole element (or group of elements). 

If we scroll, we trade the "incorrect" sampling points (which don't matter when scrolling non-integer amounts) for correct kerning. We always want to round in the y direction, as this (often) aligns the text without messing up kerning or any other downsides.